SoraFoundation/Classes/ViewModel/Preprocessors/InputViewModelPreprocessor.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+public protocol InputViewModelPreprocessing {
```

Comment:
No docs.
I suggest to name it `TextPreprocessing`

Reply:
Done.